lib/improver/tests/threshold/test_BasicThreshold.py

```diff
@@ -195,23 +231,14 @@ def test_metadata_changes(self):
         expected_attribute = "above"
         expected_units = 1
         expected_coord = DimCoord(np.array([0.1], dtype=np.float32),
-                                  long_name='threshold',
+                                  standard_name=self.cube.name(),
+                                  var_name='threshold',
                                   units=self.cube.units)
         self.assertEqual(result.name(), expected_name)
         self.assertEqual(result.attributes['relative_to_threshold'],
                          expected_attribute)
         self.assertEqual(result.units, expected_units)
-        self.assertEqual(result.coord('threshold'),
-                         expected_coord)
-
-    def test_threshold_dimension_added(self):
```

Comment:
Why has this test been removed?

Reply:
I think this is because the `process` method no longer adds a coordinate, as this is dealt with by a separate method.

Reply:
Ok